foreman-installer/foreman-installer.spec

```diff
@@ -58,11 +58,14 @@ rake install \
   SYSCONFDIR=%{buildroot}%{_sysconfdir} \
   --trace
 
+%post
+foreman-installer --scenario foreman --migrations-only > /dev/null
+
 %files
 %defattr(-,root,root,-)
 %doc README.* LICENSE
-%config %attr(600, root, root) %{_sysconfdir}/foreman/%{name}.yaml
-%config(noreplace) %attr(600, root, root) %{_sysconfdir}/foreman/%{name}-answers.yaml
+%config %attr(600, root, root) %{_sysconfdir}/foreman/installer-scenarios.d/foreman.yaml
```

Comment:
This package needs to install and own the installer-scenarios.d directory as it isn't owned by anything else.  Ideally perhaps it'd be under /etc/foreman-installer/scenarios.d/ to match the log file layout?